src/material-experimental/mdc-snack-bar/snack-bar-container.scss

```diff
@@ -13,6 +13,10 @@
   @include cdk-high-contrast(active, off) {
     border: solid 1px;
   }
+
+  &:not(._mat-animation-noopable) {
```

Comment:
I think this is cleaner, but my concern is that the `:not` increases the specificity and it can make it difficult for people to override the styles.

Reply:
True - I'd rather not add the need for more specificity for the general case with animations. Changed to be straightforward with removing `transition`. The only danger is if MDC changes the animated styles but we can deal with that another time